system-int-test/src/cmr/system_int_test/utils/association_util.clj

```diff
@@ -17,7 +17,7 @@
    (associate-by-concept-ids token concept-id coll-concept-ids nil))
   ([token concept-id coll-concept-ids options]
    (let [options (merge {:raw? true :token token} options)
-         response (transmit-variable/associate-variable
+         response (transmit-assoc/associate-concept
```

Comment:
Notice this is still calling a transmit lib function, but this is only used in tests and I think it makes sense here as after all, transmit lib provides helper functions to make http request calls easy.